openbb_platform/extensions/quantitative/openbb_quantitative/quantitative_router.py

```diff
@@ -494,12 +517,14 @@ def quantile(
     """
     import pandas_ta as ta  # pylint: disable=import-outside-toplevel # type: ignore
 
-    df = basemodel_to_df(data)
+    df = basemodel_to_df(data, index=index)
     series_target = get_target_column(df, target)
 
     df_median = ta.median(close=series_target, length=window).to_frame()
     df_quantile = ta.quantile(series_target, length=window, q=quantile_pct).to_frame()
-    results = pd.concat([df_median, df_quantile], axis=1).dropna()
+    results = (
+        pd.concat([df_median, df_quantile], axis=1).dropna().reser_index(drop=False)
```

Comment:
`reser_index()`.  I guess codespell doesn't catch these kinds of spelling typos.

Reply:
surprised other linters doesnt pick it up